utils/monitor_soap.sh

```diff
@@ -0,0 +1,250 @@
+        sudo wget http://prdownloads.sourceforge.net/staf/STAF3426-linux-amd64.tar.gz && \
+        tar -xvf STAF3426-linux-amd64.tar.gz
+    else
+        # Download STAF tar for RHEL systems
+        sudo yum update
+        sudo yum install -y openjdk-8-jdk
+        sudo wget https://sourceforge.net/projects/staf/files/staf/V3.4.26/STAF3426-zlinux-64.tar.gz && \
+        tar -xvf STAF3426-linux.tar.gz
+    fi
+    
+    update-alternatives --list java
+    update-java-alternatives -s $(update-java-alternatives -l | grep '1\.8' | cut -d " " -f1) || echo '.'
+    java -version
+
+    # Install STAF
+    cd "$HOME/staf" || exit 1;
+    echo -e "\n\n\n\n\n\n\n" | sudo ./STAFInst
+    if [[ "$?" -eq 0 ]]; then
+        echo "STAF installation successful!"
+    else
+        echo "ERROR: Cannot install STAF." && exit 1;
+    fi
+}
+
+function set_staf {
+    sudo rm -rf "$dir_qa"; sudo mkdir "$dir_qa"
+    sudo tar -xvf "$dir_bin/soapdata.tar" -C "$dir_qa" 1> /dev/null || exit 1;
+    # Edit /opt/qa/soapvalidator/conf/global.properties file, change domain/server values.
+    set_properties
+    # Register new staf services
+    echo "Adding services to SOAP"
+    staf local service add service SOAP LIBRARY JSTAF EXECUTE $dir_qa/soapvalidator/bin/zimbrastaf.jar; \
+    staf local service add service LOG LIBRARY STAFLog; \
+    staf local service add service INJECT LIBRARY JSTAF EXECUTE $dir_qa/soapvalidator/bin/zimbrainject.jar;
+    ret_code="$?"
+    if [[ "$ret_code" -eq 49 ]]; then
+        echo "Services already present in STAF, ready to execute test cases!" && \
+        staf local service list
+    elif [[ "$ret_code" -eq 0 ]]; then
+        echo "Services added to STAF, ready to execute test cases!" && \
+        staf local service list
+    else
+        echo "WARNING: Error while adding the services to STAF, test cases might not get executed!"
+    fi
+
+    echo "Setting up zimbra services. Adding services to the STAF"
+    sudo su - zimbra -c 'zmlocalconfig -e allow_unauthed_ping=true'
+    echo "Restarting Mailbox..." 
+    sudo su - zimbra -c 'zmmailboxdctl restart' || exit 1;
+}
+
+function set_properties {
+    # Edit /opt/qa/soapvalidator/conf/global.properties file, change domain/server values.
+    echo "Setting global properties."
+    domain_name="$HOSTNAME"
+    # edit with sed
+    sudo sed -i "s|localhost|$HOSTNAME|g" $dir_qa/soapvalidator/conf/global.properties
+    sudo sed -i "s|@zimbra\.com|@$domain_name|g" $dir_qa/soapvalidator/conf/global.properties
+    sudo sed -i "s|defaultdomain\.name=.*|defaultdomain.name=$domain_name|g" $dir_qa/soapvalidator/conf/global.properties
+    sudo sed -i "s|admin\.password=.*|admin.password=$ADMIN_PASS|g" $dir_qa/soapvalidator/conf/global.properties
+    sudo sed -i "s|admin\.user=.*|admin.password=$ADMIN_USER|g" $dir_qa/soapvalidator/conf/global.properties
+}
+
+function run_tests {
+    # Execute SOAP tests on the server
+    STAF LOCAL soap \
```

Comment:
I think for full test suite we need to use `nohup` to run it in the background.

Reply:
Added nohup command to run tests in the backgroud!

Reply:
Hi @umagmrit sir, By remotely triggering the execution and monitor script with `&` seems to be working well here.
Checkout - http://144.24.97.112:8080/job/deploy-dev/91/console